ui/src/features/webhooks.js

```diff
@@ -12,7 +12,7 @@ import Snackbar from "material-ui/Snackbar";
 
 
 const DESCRIPTION = 'Webhooks are events that notify you on test progress.\n' +
-    'Webhooks are supported in Slack or JSON format for an easy server to server integration.\n' +
+    'Webhooks are supported in Slack, MS Teams, or JSON format for an easy server to server integration.\n' +
```

Comment:
"supported in Slack, Microsoft Teams, or JSON" 😄

Reply:
Corrected.  Please consider adding registered/trademarked symbols to every reference of Slack and Microsoft in the future.  That is the proper way when referencing these commercial organizations.  But that is outside the scope of this feature request.
Fixed with commit 6719dd2a4a5e2d612b2793d2536f54afec2b6477